src/families/tron/synchronisation.test.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+
+import { getCryptoCurrencyById } from "../../currencies";
+import { syncAccount } from "../../__tests__/test-helpers/bridge";
+import bridge from "./bridge/js";
+
+import type { TronResources } from "./types";
+
+const tron = getCryptoCurrencyById("tron");
+const defaultSyncConfig = {
+  paginationConfig: {},
+  blacklistedTokenIds: [],
+};
+
+const makeDummyAccount = () => ({
+  type: "Account",
+  id: "",
+  derivationMode: "",
+  seedIdentifier: "",
+  name: "Tron",
+  starred: false,
+  used: false,
+  currency: tron,
+  index: 0,
+  freshAddress: "",
+  freshAddressPath: "",
+  swapHistory: [],
+  freshAddresses: [],
+  blockHeight: 0,
+  balance: BigNumber(0),
+  spendableBalance: BigNumber(0),
+  operationsCount: 0,
+  operations: [],
+  pendingOperations: [],
+  unit: tron.units[0],
+  lastSyncDate: new Date(0),
+  creationDate: new Date(),
+  balanceHistoryCache: {},
+});
```

Comment:
It seems you don't need a function for that, you can just define the data as a constant as well.

Reply:
Yeah I know, I tried to copy what Gaetan did [here](https://github.com/LedgerHQ/ledger-live-common/blob/ab7c58f65e261d37be39cc8ebe7ea59ddbf44912/src/families/tezos/synchronisation.test.js#L69) in the case at some point we would need to add parameters to create customisable accounts. 

I'll remove it for now and we'll add it later if needed 👍